projects/core/src/cms/services/cms-component.adapter.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+import { CmsComponent } from '../../occ/occ-models/cms-component.models';
+
+@Injectable({
+  providedIn: 'root',
```

Comment:
do we need it here?

Reply:
Angular DI complains when I leave it out: 

`NullInjectorError: No provider for CmsComponentAdapter!`